generate_leaf_chart.m

```diff
@@ -0,0 +1,33 @@
+% Seperate from the main functionality of the program, this script
+% generates a chart for the flow characterization of the test sprinkler
+
+clear;
+[a, r2] = sprinkler_1_data();
+flow_func = @(h) a * h.^0.5;
+parameters;
+
+p_orig = (5:2.5:35); % Pressure, psi
+v_dot_orig = [6.83, 7.77, 8.75, 9.53, 10.30, 10.82, 11.87, 12.29, ...
```

Comment:
Where do these numbers come from in v_dot_orig?

Reply:
Those are from me buying an impulse sprinkler similar to the ones to be used in Quesimpuco and measuring the volumetric throughput at various pressure levels.